bin/11_2/src/Main.cpp

```diff
@@ -12,18 +12,13 @@ bool IsSeparator(const char symbol) {
 
 // A word is a set of characters separated from the
 // rest by a set of separator characters.
-bool IsWordFound(const std::string_view word, const std::string_view line) {
-  auto begin = line.find(word);
-  if (begin == std::string_view::npos) {
-    return false;
-  } else {
-    auto end = begin + word.size() - 1;
-    // The case when a word occupies the entire line.
-    if (begin == 0 && end == line.size()) { return true; }
-
-    return IsSeparator(line.at(begin - 1))
-      && IsSeparator(line.at(end + 1));
-  }
+bool IsWordFound(const std::string& word, const std::string& line) {
+  const std::string outlinedLine { ' ' + line + ' ' };
```

Comment:
It's not a good idea to create a copy of the line, due to performance restrictions.
It would be better to change the algorithm of finding.

if ((begin == 0 || IsSeparator(begin - 1)) &&
    (end == line.size() - 1 || IsSeparator(line.at(end +1))))